sys/key_rev_large_value.js

```diff
@@ -2,14 +2,14 @@
 
 var P = require('bluebird');
 var uuid = require('cassandra-uuid').TimeUuid;
-var mwUtil = require('../lib/mwUtil');
-var HTTPError = require('../lib/exports').HTTPError;
-var URI = require('swagger-router').URI;
 var preq = require('preq');
 
-var yaml = require('js-yaml');
-var fs = require('fs');
-var spec = yaml.safeLoad(fs.readFileSync(__dirname + '/key_rev_value.yaml'));
+var mwUtil = require('../lib/mwUtil');
+var HyperSwitch = require('hyperswitch');
```

Comment:
Minor nit: I'd prefer to first have the require list of modules, and then requires of local files. Mixing them kind of decreases readability.